examples/query_lang/credentials_alternatives.json

```diff
@@ -0,0 +1,76 @@
+{
+  "credentials": [
+    {
+      "id": "pid",
+      "format": "vc+sd-jwt",
```

Comment:
let's not use media types - especially unregistered ones - please

Reply:
that is the value currently defined in the OID4VP specification and this PR is not the place to change it. please open an issue and we will discuss it, but it is out of scope for this PR.